### PR TITLE
Compute

### PR DESCRIPTION
**Pull request by lewismarshall**
Creates user-data for compute nodes

### CODE REVIEW

File: pkg/userdata/userdata.go [hunk 1]
```diff
@@ -317,20 +317,144 @@ write_files:
...
+      EnvironmentFile=/etc/environment
+
+      # Generate / check keto-token env (only needed until we update keto-tokens)...
+      ExecStartPre=/usr/bin/mkdir -p /etc/kubernetes
+      ExecStartPre=/usr/bin/touch /etc/kubernetes/keto-token.env
+      ExecStartPre=/usr/bin/docker run \
+        --rm \
+        --net host \
+        -v /etc/kubernetes/:/etc/kubernetes/ \
+        quay.io/ukhomeofficedigital/kmm \
```
Comment (jaykeshur) [inline pkg/userdata/userdata.go]: update image name to be same as repo name

Comment (lewismarshall) [inline pkg/userdata/userdata.go]: Will do but it's in a few places so I'll do that as a separate pr if that's OK

Comment (lewismarshall) [inline pkg/userdata/userdata.go]: @jaykeshur Will do but separately (it's in a few places separate from this PR)

File: pkg/userdata/userdata.go [hunk 2]
```diff
@@ -317,20 +317,144 @@ write_files:
...
+  permissions: 0644
+  owner: root
+  content: |
+    vm.max_map_count=262144
 `
 
 	data := struct {
-		KubeVersion string
-		KubeAPIURL  string
+		ClusterName			string
```
Comment (lewismarshall) [inline pkg/userdata/userdata.go]: Sorted with another push

**Comment by lewismarshall** [comment]
@vaijab Updated the ingress for masters.

**Pull request merged**